scripts/generateAssetData/baseAssets.ts

```diff
@@ -53,7 +53,7 @@ export const dogecoin: AssetWithNetworkName = {
   networkName: 'Dogecoin',
   precision: 8,
   color: '#FFC107',
-  icon: 'https://assets.coincap.io/assets/icons/256/doge.png',
+  icon: 'https://assets.coingecko.com/coins/images/5/large/dogecoin.png?1696501409',
```

Comment:
TIL! Looks weird, but is the correct icon 🤯: https://dogecoin.com/trademarks/

Reply:
Kept as it is